src/apps/AppCenter/AppCard.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Card,
+  GU,
+  IconExternal,
+  Tag,
+  textStyle,
+  unselectable,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import AppIcon from '../../components/AppIcon/AppIcon'
+
+const AppCard = React.memo(function AppCard({
+  onClick,
+  link,
+  icon,
+  name,
+  tag,
+  description,
+}) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+  const handleClick = useCallback(() => onClick(link), [onClick, link])
+
+  return (
+    <Card onClick={onClick} css={onClick ? 'display: block;' : ''}>
```

Comment:
Hmm, why do we need to change to `display: block` if an `onClick` is present?

Reply:
Fixed here: https://github.com/aragon/aragon/pull/936/commits/5360fff41b03df5bdfe67855725c5b75930aaf7e
It is from a PR that uses this as base

Reply:
As to the why is this setting `display: block` it is because of the way buttons handle updating display (as described [here](https://stackoverflow.com/questions/35464067/flexbox-not-working-on-button-or-fieldset-elements)), when the `onClick` handler is set then a `button` will be the container. This makes it so that the rendering of the inner elements works.